prisma/migrations/20220708143710_updated_schema/migration.sql

```diff
@@ -1,27 +0,0 @@
--- CreateTable
```

Comment:
This is the migration that has already happened. Migrations that have been run should not be deleted, please undo this change.